Component/SchemaFilter.php

```diff
@@ -143,7 +148,8 @@ public function getDataStoreConfig(Element $element, $schemaName)
         $elementConfig = $element->getConfiguration();
         $schemaConfigs = $elementConfig['schemes'];
         $storeConfigs = DataStoreUtil::configsFromSchemaConfigs($this->registry, $schemaConfigs);
-        return $storeConfigs[$schemaName];
+        $schemaConfig = $schemaConfigs[$schemaName];
+        return $storeConfigs[$schemaConfig["featureType"]];
```

Comment:
Is the key `featureType` guaranteed to always be present?

Reply:
I am not sure, but schemaName is definitely wrong.  What happens here is that a pre-defined Featuretype from a yml file in the repo is included in the config - based on its name. It can happen though, that the Featuretype is already defined in the config itself instead of taken from a yml file. I did not test it, but that should not make a difference for the changed code. To my knowledge, the reason for this bug might have been that Schemas and Featuretypes are coincidentely given the same names in some configuration, which made the code work by chance.